features/unsupported/tests/usb/host/mass_storage/main.cpp

```diff
@@ -1,42 +1,66 @@
 #include "mbed.h"
 #include "USBHostMSD.h"
 DigitalOut led(LED1);
-void msd_task(void const *) {
-	printf("init msd\n");
-	USBHostMSD msd("usb");
-	int i = 0;
-	printf("wait for usb memory stick insertion\n");
-	while(1) {
-
-		// try to connect a MSD device
-		while(!msd.connect()) {
-			Thread::wait(500);
-		}
-
-		// in a loop, append a file
-		// if the device is disconnected, we try to connect it again
-
-		// append a file
-		FILE * fp = fopen("/usb/test1.txt", "a");
-
-		if (fp != NULL) {
-			fprintf(fp, "Hello fun SD Card World: %d!\r\n", i++);
-			printf("Goodbye World!\r\n");
-			fclose(fp);
-		} else {
-			printf("FILE == NULL\r\n");
-		}
-		Thread::wait(500);
-		printf("again\n");
-		// if device disconnected, try to connect again
-		while (msd.connected()) {
-			Thread::wait(500);
-		}
-	}
+#include "FATFileSystem.h"
+#include <stdlib.h>
+
+
+void msd_task(void const *)
+{
+
+    USBHostMSD msd;
+    int i = 0;
+    FATFileSystem fs("usb");
+    int err;
+    printf("wait for usb memory stick insertion\n");
+    while(1) {
+
+        // try to connect a MSD device
+        while(!msd.connect()) {
+            Thread::wait(500);
+        }
+        if (fs.mount(&msd) != 0) continue;
```

Comment:
Before pushing the patch I have used the astyle.exe command from page https://docs.mbed.com/docs/mbed-os-handbook/en/5.1/cont/code_style/ 
To fix the above  indentation issue I added option -j .
 AStyle.exe --style=kr --indent=spaces=4 --indent-switches -j
This solves the following requirement.
•One true brace style (1TBS) - use braces for statements of type if, else, while and for (exception from K&R).  


-j is taking into account the above remarks.